### DISCUSSION TITLE
not able to use Allure report with Reqnroll : AI_FAIL Aspect injector processing has failed , not able to see the report

### DISCUSSION THREAD

**Discussion by gabrielaiobogeanu** [reqnroll/Reqnroll#257] [Q&A]
Hi, unfortunately we are not able to use allure with Reqnroll, we followed the steps correctly. Maybe someone can help , it will be highly appreciated.

Our problem is the following:
With a clean project and following the documentation (https://allurereport.org/docs/reqnroll/) after the package is added on step 1.3 we are getting the following error on the project where we added the package:
![imagem (1)](https://github.com/user-attachments/assets/c5275673-5465-4124-b161-fbac269e1c46)

If we build the project on admin mode then we don't get this error but after the tests are ran, and the allure generate and
allure open are executed, the start up service with the report shows 0 tests

**Comment by gasparnagy** [comment]
Just to clarify... The Step 1.3 in the referred doc is

```
dotnet add ⟨PATH TO PROJECT⟩ package Allure.Reqnroll
```

Is that what causes you the error? Is that error coming when you build?

**Reply by gabrielaiobogeanu** [reply]
yes, the error starts right after the package is added

**Reply by gasparnagy** [reply]
I see. From the error message, I guess that your project uses [Aspect Injector](https://github.com/pamidur/aspect-injector) library and that has some compatibility problem with the Reqnroll Allure plugin. 
I suggest asking the [Aspect Injector team](https://github.com/pamidur/aspect-injector/issues) about how can you get a more detailed error message and based on that it will be easier to find the root cause.

**Reply by gabrielaiobogeanu** [reply]
thank you for your reply. 
We double checked and we don't use the Aspect Injector library...
But we saw that allure adds the aspect injector 
![imagem (3)](https://github.com/user-attachments/assets/33965fe2-8d98-4cbe-8a9c-3e6f6f305cea)
![imagem (2)](https://github.com/user-attachments/assets/c3d61c13-dad1-4991-b3af-40600940b93c)

**Reply by gabrielaiobogeanu** [reply]
we are installing the Allure reqnroll that it's using the Allure.net.commons and in there the AspectInjector is used

**Comment by MerlinPonrose** [comment]
Hi Team,

@gasparnagy  @gabrielaiobogeanu 

I am also facing same error "Aspect Injector processing has failed" when trying to install Allure.reqnroll.

I am not having admin access in my machine will that cause this issue.

If so kindly suggest on resolving the error.
